Launcher/Classes/ViewControllers/DeviceSettingsViewController.swift

```diff
@@ -4,19 +4,23 @@ class DeviceSettingsViewController: NSViewController {
     let applicationStateHandler = ApplicationStateHandler()
 
     @IBOutlet weak var textField: NSTextField!
+    @IBOutlet weak var bundleID: NSTextField!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         if let deviceIP = applicationStateHandler.deviceIP {
             textField.stringValue = deviceIP
         }
+        if let bundleIDdata = applicationStateHandler.bundleID {
```

Comment:
Why do you bind this to `bundleIDdata` and not just `bundleID`?

Reply:
Because bundleID is already an outlet name

Reply:
Ah, right. I think it would be more readable when we rename the `bundleID` outlet to `bundleIDTextField` (and also rename the vanilla `textField`; what's that?) and then in turn rename the `bundleIDdata` to `bundleID`.

Reply:
Done. Don't like `bundleIDTextField: NSTextField` tho. But this is always better than names before ;)